extensions/amp-app-banner/0.1/amp-app-banner.js

```diff
@@ -397,6 +397,8 @@ export class AmpAndroidAppBanner extends AbstractAppBanner {
  */
 function handleDismiss(state) {
   hideBanner(state);
+  // Restore focus to document after dismiss for screen readers.
+  state.element.ownerDocument.body./*REVIEW*/focus();
```

Comment:
Actually thinking about this some more, this would only address it when we are actually using fixed layer which is currently only kicks off on iOS. @dvoytenko to confirm my assumption. What do you think?